coops-ui/tests/unit/ARDate.spec.ts

```diff
@@ -0,0 +1,36 @@
+import Vue from 'vue'
+import Vuetify from 'vuetify'
+import Vuelidate from 'vuelidate'
+
+import store from '@/store/store'
+import { shallowMount } from '@vue/test-utils'
+import ARDate from '@/components/AnnualReport/BCorp/ARDate.vue'
+
+Vue.use(Vuetify)
+Vue.use(Vuelidate)
+
+describe('AnnualReport - Part 1 - UI', () => {
+  beforeEach(() => {
+    // init store
+    store.state.nextARDate = '2020-09-18T23:15:53.785045+00:00'
+  })
+
+  it('initializes the store variables properly', () => {
+    const wrapper = shallowMount(ARDate, { store })
+    const vm: any = wrapper.vm
+
+    expect(vm.$store.state.nextARDate).toEqual('2020-09-18T23:15:53.785045+00:00')
+    wrapper.destroy()
+  })
+
+  it('succeeds when the Annual report date outputs are correct', () => {
+    const wrapper = shallowMount(ARDate, { store })
+    const vm: any = wrapper.vm
+    const regex = / (?!.* )/
+    const today = new Date().toDateString().split(' ').slice(1).join(' ').replace(regex, ', ')
+
+    expect(vm.$el.querySelector('.ar-date').textContent).toContain('Sep 18, 2020')
```

Comment:
Severin: Does this test pass today (Sep 24)?
To fix this, instead of doing new Date(), you can create a specified date and then check that the element contains it.

Cameron: Certainly, that change could be made! This field however validates against the current day and is unique everyday, by hardcoding the input and output were not exactly testing the function behind the calculation. What are your thoughts?

Severin: I was thinking you'd do something like `new Date(2020, 09, 18, 23, 15, 53)`, which still tests the function.

Reply:
Not sure i entirely follow this recommendation, the method behind that field is that of the date-mixins `toReadableDate`, of which the argument passed is of a specific format returned from the back end and the output is specific to the return of that method. I'm certainly open to adjusting this, but i'm not entirely sure what you're recommending.

Reply:
On line 30, initialize date as per my example.

Reply:
If we hard code the `today` date, it no longer tests that todays date is the output & in the correct format. The purpose of this test is two-fold,
1) It is `today's` date.
2) It is output in the correct format.

I'm not saying the test is perfect by any means, but by hard coding a specific date, it fails the test, as that field expects todays date.  Maybe there is a little confusion as per the `nextARDate`? That date is predefined and validated to ensure the output is the correct format, whereas today's date is dynamic.

Reply:
Oops! Mea culpa. I was looking at wrong date.